src/store/paymentMethods.ts

```diff
@@ -0,0 +1,47 @@
+import { defineStore } from "pinia";
+
+import { type Pagoplux, type PaymentMethods } from "@/interfaces/paymentMethods.interface";
```

Comment:
abstract the type word and add it next to 'import',  types are at the end of the imports after a breakline

Reply:
okik